tests/tasks/tasks.test.ts

```diff
@@ -784,4 +785,14 @@ describe('CONTRACT TASKS', () => {
             throw new Error(`Error in deregestering provider: ${error}`)
         }
     })
+
+    it.only('Calculate captcha solution on the basis of Dapp users provided solutions', async () => {
+        const providerTasks = new Tasks(mockEnv)
+        try {
+            const result = await providerTasks.calculateCaptchaSolutions()
+            should().equal(result, undefined)
```

Comment:
This needs to be better tested. How do we know that the calculated Captcha Solutions are actually correct? There should be a check of the before and after result.

Reply:
Updated the test case by checking the number of captchas with solution before and after.